package.json

```diff
@@ -101,22 +101,23 @@
     "@flybywiresim/api-client": "^0.16.2",
     "@flybywiresim/react-components": "^0.3.1",
     "@flybywiresim/tailwind-config": "^0.5.0",
+    "@localazy/cli": "^1.6.0",
     "@reduxjs/toolkit": "^1.6.2",
     "@rollup/plugin-json": "^4.1.0",
     "@sentry/browser": "^6.17.7",
     "@sentry/integrations": "^6.17.7",
     "@sentry/tracing": "^6.17.7",
     "@tabler/icons": "^1.41.2",
     "@types/react-canvas-draw": "^1.1.1",
-    "@localazy/cli": "^1.6.0",
+    "@types/semver": "^7.3.13",
```

Comment:
This should be in `devDependencies`, since it's not a runtime package.

Reply:
fixed